app/views/questions/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @question do |f| %>
+  <%= f.text_field :title, placeholder: "Title" %>
+  <%= f.text_area :body, placeholder: "Body" %>
+  <%= f.label :category %>
+  <%= f.collection_select(:category_id, @categories, :id, :name, class: "browser-default") %>
```

Comment:
This is the dropdown that we cannot see in the view, but we can see rendered in HTML when we inspect. We are using Materialize to style, if that's relevant.